runway/hooks/staticsite/auth_at_edge/client_updater.py

```diff
@@ -0,0 +1,117 @@
+"""User Pool Client Updater.
+
+Responsible for updating the User Pool Client with the generated
+distribution url + callback url paths.
+"""
+
+import logging
+# pylint: disable=unused-import
+from typing import Any, Dict, Optional
+
+import boto3
+
+from ....cfngin.lookups.handlers.output import OutputLookup
+from ....cfngin.lookups.handlers.rxref import RxrefLookup
+# pylint: disable=unused-import
```

Comment:
this should be in-line so it does not apply to the entire file, only a single line